ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Defaults.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+
+object Defaults {
+
+  val LedgerClockGranularity: FiniteDuration = 1.second
+
+  val TimeoutScaleFactor: Double = 1.0
+
+}
```

Comment:
It probably makes sense to keep these in the same file as `Config` (it's used only there, right?)

Reply:
_Config_ is not accessible from _Tests_, unfortunately, and it's used there too.